.github/workflows/test.yml

```diff
@@ -10,30 +10,39 @@ jobs:
           npm install
       - run: |
           npm run all
+        env:
+          ACTIONS_ALLOW_UNSECURE_COMMANDS: ${{ secrets.ACTIONS_ALLOW_UNSECURE_COMMANDS }} 
```

Comment:
Having some trouble tracking down the documentation for this variable. Can you provide context on what this is doing?

Reply:
Currently this error is thrown:
`Unable to process command '::add-path::/opt/hostedtoolcache/waypoint/0.1.0/x64' successfully.`

> The `add-path` command is disabled. Please upgrade to using Environment Files or opt into unsecure command execution by setting the `ACTIONS_ALLOW_UNSECURE_COMMANDS` environment variable to `true`. For more information see: https://github.blog/changelog/2020-10-01-github-actions-deprecating-set-env-and-add-path-commands